ui/k-toolbar/main.kytos

```diff
@@ -1,26 +1,36 @@
 <template>
-    <k-toolbar-item icon="gear" tooltip="Maintenace">
-      <k-accordion>
-        <k-accordion-item title = "Create Maintenance Window" :key = "componentKey">
-                <k-input placeholder = "Id" :value.sync ="id"></k-input>
-                <k-input placeholder = "Description" :value.sync = "description"></k-input>
-                <k-input placeholder = "Start time" :value.sync = "start"></k-input>
-                <k-input placeholder = "End time" :value.sync = "end"></k-input>
-                <k-select icon = "link" title = "List of Switches" :options = "dpid_names" :value.sync = "items"></k-select>
-                <k-select icon = "link" title = "List of Interfaces" :options = "interface_names" :value.sync = "interface_items"></k-select>
-                <k-select icon = "link" title = "List of Links" :options = "link_names" :value.sync = "link_items"></k-select>
-                <k-button title = "Reset Fields" :on_click = "forceRenderer"></k-button>
-                <k-button title = "Create Maintenance Window" :on_click = "post_window"></k-button>
-        </k-accordion-item>
-         <k-accordion-item title = "List Maintenance Windows">
-                <k-button tooltip="Maintenance Windows"
-                          title="List Maintenance Windows"
-                          icon="desktop"
-                          :on_click="list_windows">
-                </k-button>
-         </k-accordion-item>         
-     </k-accordion>
-    </k-toolbar-item>
+  <k-toolbar-item icon="gear" tooltip="Maintenace">
+    <k-accordion>
+      <k-accordion-item title = "Create Maintenance Window" :key = "componentKey">
+        <k-input placeholder = "Id" :value.sync ="id"></k-input>
+        <k-input placeholder = "Description" :value.sync = "description"></k-input>
+        <k-input placeholder = "Start time" :value.sync = "start"></k-input>
+        <k-input placeholder = "End time" :value.sync = "end"></k-input>
+
+        <k-select icon = "link" title = "List of Switches"
+          :options = "dpid_names" :value.sync = "switches">
+        </k-select>
+
+        <k-select icon = "link" title = "List of Interfaces"
+          :options = "interface_names" :value.sync = "interfaces">
+        </k-select>
+
+        <k-select icon = "link" title = "List of Links"
+          :options = "link_names" :value.sync = "links">
+        </k-select>
+        
+        <k-button title = "Reset Fields" :on_click = "forceRenderer"></k-button>
+        <k-button title = "Create Maintenance Window" :on_click = "post_window"></k-button>
+      </k-accordion-item>
+      <k-accordion-item title = "List Maintenance Windows">
+        <k-button tooltip="Maintenance Windows"
+          title="List Maintenance Windows"
+          icon="desktop"
+          :on_click="list_windows">
```

Comment:
David, if you could review the JS changes, in the UI it's hanging when listing existing windows, if you could fix and double check:

![20221219_115714](https://user-images.githubusercontent.com/1010796/208456015-39c56382-9882-4af7-84a1-96de9a6f73ff.png)